### DISCUSSION TITLE
How to query the transaction detail which have the internal transfer by rpc api?

### DISCUSSION THREAD

**Discussion by carrie143** [filecoin-project/lotus#12696] [Developers/Service Providers]
1.I want to query the internal transfer detail in transaction,like:
https://filscan.io/message/bafy2bzacecy4adusijbg6qdjx6w7aifgkbrkgst2ovu42zghrupgy22g5sek6/#trade
There is a internal transfer,send 1000fil from f02260507 to f03024609
2. I try to call "Filecoin.ChainGetMessage",but don't return internal transfer detail,So any other json-rpc api to use?

**Comment by rvagg** [comment]
Marking this as closed since it's a duplicate of https://github.com/filecoin-project/lotus/issues/12697, which I left a reply on. Others are welcome to join in though if they have alternative approaches that may be more efficient than what I've outlined.
